src/models/FileSystem.js

```diff
@@ -55,6 +55,13 @@ function bindFs(target, fs) {
     else if (fs.rmdir.length > 2) target._rm = pify(fs.rmdir.bind(fs))
     else target._rm = rmRecursive.bind(null, target)
   }
+
+  // Handle the special case of `cp`
+  if (isPromiseFs(fs)) {
+    if (fs.cp) target._cp = fs.cp.bind(fs)
+  } else {
+    if (fs.cp) target._cp = pify(fs.cp.bind(fs))
+  }
```

Comment:
There is already an if statement that checks `isPromiseFs(fs)` can you put your wrappers there?